tests/foreman/cli/test_syncplan.py

```diff
@@ -365,6 +363,33 @@ def test_positive_info_enabled_field_is_displayed(self):
         result = SyncPlan.info({'id': new_sync_plan['id']})
         self.assertIsNotNone(result.get('enabled'))
 
+    @tier1
+    @upgrade
+    def test_positive_info_with_assigned_product(self):
+        """Verify that sync plan info command returns list of products that
+        assigned to that sync plan
+
+        :id: 7a7e5e40-7776-4d26-9173-73f00de6e8e9
+
+        :expectedresults: Expected product information is returned in info
+            command
+
+        :BZ: 1390545
+        """
+        sync_plan = self._make_sync_plan({
+            'enabled': 'false',
```

Comment:
nitpick: you could use bool `False` instead of that string

Reply:
getting exception on that, it seems that it is necessary change factory method signature or something. Anyway, skip at that moment